src/main/java/racingcar/model/Winners.java

```diff
@@ -3,28 +3,40 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class Winners {
-	private List<String> winners;
+import racingcar.model.dto.WinnerDTO;
+
+public final class Winners {
+	private List<Car> winners;
 
 	public Winners() {
 		winners = new ArrayList<>();
 	}
 
-	public void findWinners(Cars cars) {
-		int maxDistance = 0;
-		for (int carNum = 1; carNum <= cars.size(); carNum++) {
-			if (cars.car(carNum).distance() == maxDistance) {
-				winners.add(cars.car(carNum).name());
-			}
-			if (cars.car(carNum).distance() > maxDistance) {
-				winners = new ArrayList<>();
-				winners.add(cars.car(carNum).name());
-				maxDistance = cars.car(carNum).distance();
-			}
+	public List<Car> winners() {
+		return winners;
+	}
+
+	public void findWinners(final Cars cars) {
```

Comment:
```suggestion
	public List<Car> findWinners() {
	}
```

메서드가 find이므로 우승자 자동차 목록을 반환하도록 구현 가능함
강의 영상 참고